README.md

```diff
@@ -17,6 +17,17 @@ HTTP 1.1). The services available to it are
 This is an [Angular] appliation and written in [TypeScript] in `web/sd-ran-gui`.
 See the [README.md](web/sd-ran-gui/README.md) in that folder for more info.
 
+## Building the application
+```bash
+make all
+```
+
+Prerequisites:
+```bash
+npm i grpc-web
```

Comment:
Yes - sorry I should have told you these needed to be installed. In fact `npm install` would have done it too, as long as they were in the `package.json` - and they weren't when you first ran it. I don't know if they need to be called out here